po/pt_BR.po

```diff
@@ -25,30 +28,40 @@ msgstr ""
 #: data/io.github.mightycreak.Diffuse.desktop.in:3
 #: data/io.github.mightycreak.Diffuse.appdata.xml.in:4
 msgid "Diffuse"
-msgstr "Diffuse"
+msgstr "Comparador de Arquivos Diffuse"
 
 #: data/io.github.mightycreak.Diffuse.desktop.in:4
 #: data/io.github.mightycreak.Diffuse.appdata.xml.in:5
 msgid "Graphical tool for merging and comparing text files"
-msgstr "Ferramenta gráfica para unificar e comparar arquivos de texto"
+msgstr "O ‘Diffuse’ é uma ferramenta por interface gráfica "
+"que permite comparar e mesclar os arquivos de texto, "
+"comparar uma quantidade arbitrária de arquivos lado a "
+"lado e oferece aos usuários a capacidade de ajustar "
+"manualmente a correspondência das linha e editar os "
+"arquivos diretamente, possibilita recuperar as revisões "
+"feitas com as ferramentas ‘Bazaar’, ‘CVS’, ‘Darcs’, ‘Git’, "
+"‘Mercurial’, ‘Monotone’, ‘RCS’, ‘Subversion’ e ‘SVK’"
 
 #: data/io.github.mightycreak.Diffuse.appdata.xml.in:7
 msgid ""
 "Diffuse is a graphical tool for comparing and merging text files. It can retrieve "
 "files for comparison from Bazaar, CVS, Darcs, Git, Mercurial, Monotone, RCS, and "
 "Subversion repositories."
 msgstr ""
-"O Diffuse é uma ferramenta gráfica para unificar e comparar arquivos de texto. "
-"Também pode obter revisões para comparar arquivos de repositórios Bazaar, CVS, "
-"Darcs, Git, Mercurial, Monotone, RCS, e Subversion. "
+"O ‘Diffuse’ é uma ferramenta por interface gráfica que permite comparar e "
+"mesclar os arquivos de texto, comparar uma quantidade arbitrária de arquivos "
+"lado a lado e oferece aos usuários a capacidade de ajustar manualmente a "
+"correspondência das linha e editar os arquivos diretamente, possibilita "
+"recuperar as revisões feitas com as ferramentas ‘Bazaar’, ‘CVS’, ‘Darcs’, "
+"‘Git’, ‘Mercurial’, ‘Monotone’, ‘RCS’, ‘Subversion’ e ‘SVK’."
```

Comment:
change: you seem to have copy/pasted the same paragraph :face_with_diagonal_mouth: